fileglancer/uimodels.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional, Generic, TypeVar
+from datetime import datetime, timedelta
+from pydantic import BaseModel, Field
+
+
+# TODO:
+# These are candidates to be extracted to a shared library
+# They have all been copied from fileglancer-central/fileglancer-central/model.py
+
+class FileSharePath(BaseModel):
```

Comment:
This used to be in paths.py but I moved it to a different file in case these will be all moved to a lib later